packages/client/src/users/types/index.ts

```diff
@@ -7,3 +7,6 @@ export * from './postUserAttributes.types';
 export * from './putUserAttribute.types';
 export * from './patchUserAttribute.types';
 export * from './deleteUserAttribute.types';
+export * from './contact.types';
```

Comment:
I think instead of creating only one file for all the contacts actions, it should be divided in a few files, one file for each action. (the same applies for guestUser and Titles) For example:
```
getContact.types
postContact.types
patchContact.types
...
```
and then, for example, on your **getContact.types** you should have this:

```
export type GetContact = (
  id: number,
  contactId: string,
  query?: Query,
  config?: Record<string, unknown>,
) => Promise<ContactResponse>;
```

and that **ContactResponse** should be, like you have, in contact.types.

So, one file for each action and use your contact.types for contact response (I think it should be the same for most of the actions).
You can check other examples like returns, bags, to get an idea of what I'm saying.
But if you need help you can talk to me :)